scaffolding-chef-inspec/lib/scaffolding.ps1

```diff
@@ -2,12 +2,16 @@
 # A scaffolding for an InSpec profile
 #
 
+if(!$scaffold_inspec_client){
+    $scaffold_inspec_client = "stuartpreston/inspec"
```

Comment:
Should we default to "chef/inspec"?

Reply:
Unfortunately, we don't have a windows `chef/inspec` package yet this is being worked on at the moment.